src/static/css/pages/careers.less

```diff
@@ -0,0 +1,3 @@
+.students-and-graduates .media_image {
+    width: 150px
```

Comment:
As I understand it there are two sets of image sizes in the media images:
1. **Circle images**.130x130px up to 800px screen-width, then 150x150px.
2. **Non-circular images**. 100x100px at all screen sizes.

I've implemented `media_image__100` and `media_image__130-to-150` in https://github.com/cfpb/cfgov-refresh/pull/686. I think this PR can go ahead and we can consolidate later.